plugins/modules/database/mysql/mysql_user.py

```diff
@@ -60,6 +60,13 @@
         user instead of overwriting existing ones.
     type: bool
     default: no
+  tls_requires:
+    description:
+      - Set requirement for secure transport as a dictionary of requirements (see the examples).
+      - Valid requirements are SSL, X509, SUBJECT, ISSUER, CIPHER.
+      - SUBJECT, ISSUER and CIPHER are complementary, and mutually exclusive with SSL and X509.
+      - U(https://mariadb.com/kb/en/securing-connections-for-client-and-server/#requiring-tls).
+    type: dict
```

Comment:
```suggestion
    type: dict
    version_added: 1.0.0
```

Reply:
@felixfontein should I add this verbatim? will the version number be adjusted automatically? I don't know how that works.

Reply:
Please add this verbatim. If this PR is merged before 1.0.0 is released, this is what is needed. If it is not merged in time, the version number needs to be bumped to 1.1.0. (We'll tell you if that's the case.)